CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing Guide
+
+Before submitting your contribution, please make sure to take a moment and read through the following guidelines:
+
+- [Pull Request Guidelines](#pull-request-guidelines)
+- [Development Setup](#development-setup)
+- [Scripts](#scripts)
+- [Project Structure](#project-structure)
+- [Contributing Tests](#contributing-tests)
+
+## Pull Request Guidelines
+
+- Checkout a topic branch from a base branch, e.g. `master`, and merge back against that branch.
+
+- If adding a new feature add accompanying test case.
+
+- It's OK to have multiple small commits as you work on the PR - GitHub can automatically squash them before merging.
+
+- Make sure tests pass!
+
+- Commit messages must follow the [commit message convention](./commit-convention.md). Commit messages are automatically validated before commit (by invoking [Git Hooks](https://git-scm.com/docs/githooks) via [husky](https://github.com/typicode/husky)).
+
+- No need to worry about code style as long as you have installed the dev dependencies - modified files are automatically formatted with Prettier on commit (by invoking [Git Hooks](https://git-scm.com/docs/githooks) via [husky](https://github.com/typicode/husky))..
+
+## Development Setup
+
+You will need [Node.js](https://nodejs.org) **version 16+**.
```

Comment:
Can we use Nodejs 14+?

Reply:
@adamant-al 16 is the current LTS version of NodeJS. We can easily use 14+ to support older projects

Reply:
v14 EOL is 2023-04-30, I think we should support it.

Reply:
Oh, this version is for _development_. I think it's better to leave it like this

Reply:
Ok